packages/opc-back-to-top/src/opc-back-to-top.scss

```diff
@@ -0,0 +1,21 @@
+@import '@one-platform/opc-styles';
+
+$LOCAL: opc-back-to-top;
+
+$variables: ( 
+  //Add your variables here
+);
+
+.hide{
+  display: none;
+}
+.show {
+  font-family: 'Red Hat Text', sans-serif;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
add a line break after each class definition

Reply:
you mean there should be one line space between 2 classes. ?

Reply:
yes

Reply:
Okay